packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -83,9 +83,12 @@ function hasExpectedAssetData(
 export class QuoteRequestor {
     private readonly _rfqtMakerEndpoints: string[];
     private readonly _schemaValidator: SchemaValidator = new SchemaValidator();
+    private readonly _warningLogger: (s: string) => void;
 
-    constructor(rfqtMakerEndpoints: string[]) {
+    constructor(rfqtMakerEndpoints: string[], logger: (s: string) => void = logUtils.warn.bind(logUtils)) {
```

Comment:
instead of using. `bind`, is it more intuitive to simply do
```
logger: (s: string) => void =  (s) => logUtils.warn(s)
```

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2562/commits/980246d07a6049864ac7912cc2f65aa6421368f1